ansible/roles/influxdb/defaults/main.yml

```diff
@@ -16,4 +16,4 @@
 #
 
 influxdb_rpm: influxdb-1.8.3.x86_64.rpm
-influxdb_checksum: "sha512:4c0557c24e5083e9d94d49cfad496f45421bb50aacf75737ca6bff36c8ef276af884c5419b5cc74e4fa1e8062e6bee18af53e3650eaa8a6ecd656346151ea87c"
+influxdb_checksum: "sha512:60abf31c70beb8a391ded82569532a255384516da0b3c287d20065fe7047be343c50b37bbe31d5dd8df4bcb31144ef696b90e21d6739ce375b7e4fc33bae4f03"
```

Comment:
This is out of scope of this PR, and comes with security risks that need to be checked to ensure the RPM isn't modified in some way. Please put this in a separate PR, in case we need to discuss it. There's previous history with this. See #381

Reply:
Addressed in #412